docs/samples/Microsoft.ML.Samples/Static/LightGBMRegression.cs

```diff
@@ -1,7 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
```

Comment:
I think we don't need the license in the files in the samples folder. I don't see it in other files, maybe better to remove it for consistency.